src/client/automation/playback/scroll.js

```diff
@@ -169,16 +172,27 @@ export default class ScrollAutomation {
         return { x, y };
     }
 
+    _isChildFullyVisible (parentDimensions, childDimensions, offsetX, offsetY) {
+        const { x, y } = this._getChildPointAfterScroll(parentDimensions, childDimensions, parentDimensions.scroll.left, parentDimensions.scroll.top);
+
+        const { left, top } = this._getScrollPosition(parentDimensions, childDimensions, {
+            left: 0,
+            top:  0
+        }, offsetX, offsetY);
+
+        return !this._isTargetElementObscuredInPoint(x, y) && left === parentDimensions.scroll.left && top === parentDimensions.scroll.top;
```

Comment:
(minor)
I think it's better to divide long string like this by logic operators, e.g.:
```js
return !this._isTargetElementObscuredInPoint(x, y) &&
               left === parentDimensions.scroll.left && top === parentDimensions.scroll.top;
```